dimod/generators/fcl.py

```diff
@@ -144,6 +155,17 @@ def frustrated_loop(graph: GraphLike,
         raise RuntimeError(f"number of found cycles, {good_cycles}, is below " 
                            f"requested {num_cycles}")
 
+    if planted_solution is not None:
+        warnings.warn("planted_solution is deprecated; use spin-reversal transforms: "
```

Comment:
make sure you're catching all the warnings emitted in the unittests with `with self.assertWarns(DeprecationWarning):`, running the unittests should not emit warnings